CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# Contribution Guidelines
+## Branches
+
+* *master*:  images on the master branch are built monthly.
+* *develop*: images on this branch are when commits are pushed.
```

Comment:
If no commits are pushed to develop they are built daily by Travis cronjob.


Everything else is good. I'll merge.

Reply:
Fixed